packages/sonarwhal/docs/user-guide/further-configuration/development-flow.md

```diff
@@ -0,0 +1,67 @@
+# Development flow integration
+
+`sonarwhal` can be used at different steps of the development flow. The
+following is a guide that ilustrates some of this scenarios and how can they
+be achieved. If you have another scenario that is not in here, please open an
+issue [on sonarwhal's GitHub repo][sonarwhal github] so we can investigate how
+to support it.
+
+The recommended way to install `sonarwhal` for these scenarios is as a
+`devDependency` (`npm install sonarwhal --save-dev`) and that's what it's
+assumed here.
+
+# Test a local server
+
+In this scenario you want to run `sonarwhal` against a local server that is
+running the code you want to test. Usually the tasks to perform can be reduced
+to:
+
+1. Start server
+1. Start sonarwhal
+
+A solution to automate this is via `npm scripts` and the `test` task. The
+biggest concern is how to start both tasks and kill the other when one ends in
+a multiplatform environment. For example, the following will not work in most
+cases because `start` will never finish:
+
+```json
+{
+    ...
+    "scripts": {
+        ...
+        "test": "npm start && sonarwhal http://localhost:8080",
+        ...
+    }
+    ...
+}
+```
+
+While there are some solutions like using just `&` or `|`, these don't work in
+all platforms. To solve this issue you will need to use a package that can
+start multiple `npm` tasks simulatenous and coordinate them.
+[`npm-run-all`][npm-run-all] is one of those.
+
+As an example, the following `npm script` will build the project, start
+[`http-server`][http-server] (assuming the output of the build process is on
+the `dist` folder), run `sonarwhal` on `localhost` and once it ends, kill the
+server started by `http-server`:
+
+```json
+{
+    ...
+    "scripts": {
+        ...
+        "http-server": "http-server dist -s -g",
```

Comment:
I think you should call the script `start-server` to be consistent

Reply:
You are right, I'll make it `start`.